addons/containerd/1.5.11/install.sh

```diff
@@ -250,3 +250,75 @@ function _containerd_migrate_images_from_docker() {
         (set -x; ctr -n=k8s.io images import $image)
     done
 }
+function _yum_install_host_packages() {
```

Comment:
this function will be removed before the commit. this is just to get the testgrid to run.

Reply:
I'll approve after this func is removed. Testgrid [runs](https://testgrid.kurl.sh/run/pr-3291-3c7091f-containerd-1.5.11-k8s-docker-2022-08-16T00:48:56Z ) for this PR looks satisfactory per our thread on slack.

Reply:
Added back this code to re-run testgrid, will remove before commit.